indico/modules/events/views.py

```diff
@@ -120,9 +120,43 @@ def _getHeader(self):
     def getJSFiles(self):
         return WPDecorated.getJSFiles(self) + self._asset_env['modules_event_display_js'].urls()
 
+    def _getJsonLDData(self):
+        lddata = {
+            "@context": "http://schema.org",
+            "@type": "Event",
+            "url": self.event.external_url,
+            "name": self.event.title,
+            "startDate": self.event.start_dt.isoformat(),
+            "endDate": self.event.end_dt.isoformat(),
+            "location": {
+                "@type": "Place",
+                "name": self.event.venue_name or 'No location set',
+                "address": self.event.address or 'No address set'
+                },
```

Comment:
you like this alignment, don't you :D

Reply:
eclipse...

Reply:
ouch ;) I didn't realize anyone still uses eclipse/pydev. Might want to look at PyCharm. But I guess eclipse/pydev also has a setting for the brace alignment